templates/prescription/general_edit.html

```diff
@@ -256,10 +265,22 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<script type="text/javascript" src="{$WEBROOT}/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="{$WEBROOT}/library/js/jquery-ui.min.js"></script>
 {literal}
 <!-- for the fancy jQuery stuff -->
 <script type="text/javascript">
 
+$(function() {
+    
+    //Drug autocomplete
+    $(".drug").autocomplete({
```

Comment:
Will this compete with the original $("#drug").autocomplete({ call that is already in this script?